crates/nu-command/src/stor/insert.rs

```diff
@@ -11,14 +11,14 @@ impl Command for StorInsert {
 
     fn signature(&self) -> Signature {
         Signature::build("stor insert")
-            .input_output_types(vec![(Type::Nothing, Type::table())])
+            .input_output_types(vec![(Type::Any, Type::table())])
```

Comment:
Shouldn't the input type be record instead of any?

Reply:
Ideal it would be but I didn't see a way to specify that it's a record without also specifying how many fields the record has. Not sure if I'm just missing something.

Reply:
I think you can do it this way `(Type::record(), Type::table())`

Reply:
```
> crates/nu-command/src/stor/insert.rs:14:39
   |
14 |             .input_output_types(vec![(Type::Record(), Type::table())])
   |                                       ^^^^^^^^^^^^-- an argument of type `Box<[(std::string::String, nu_protocol::Type)]>` is missing
   |
note: tuple variant defined here
  --> /Users/bobbypalmer/projects/nushell/crates/nu-protocol/src/ty.rs:31:5
   |
31 |     Record(Box<[(String, Type)]>),
   |     ^^^^^^
help: provide the argument
   |
14 |             .input_output_types(vec![(Type::Record(/* Box<[(std::string::String, nu_protocol::Type)]> */), Type::table())])
   |                                           
```
This was the result of trying to compile it with Type::Record(). Not sure if there's some other constructor.

Reply:
Type::record() not Type::Record()